build-libcudf.xml

```diff
@@ -41,7 +41,6 @@
       <!-- benchmarks in cudf have to be built in order to get the library used
            for data generation to build in cudf so that benchmarks in
            spark-rapids-jni can link to it. -->
-      <arg value="-DBUILD_BENCHMARKS=${BUILD_BENCHMARKS}"/>
```

Comment:
This option was not just to avoid building cudf benchmarks but also building the benchmarks in general.  It would be nice to be able to avoid building benchmark code when just building a new spark-rapids-jni jar locally, but I would advocate that we default this to true (or at least have the CI jobs always enable building the benchmarks).

Reply:
Ah, nevermind, misread this.  SPARK_RAPIDS_JNI_BUILD_BENCHMARKS still exists, and this was just the option passed to libcudf.